ui/desktop/app/templates/scopes/scope/projects/targets/target/hosts.hbs

```diff
@@ -26,7 +26,13 @@
               <code>{{host.id}}</code>
             </Copyable>
           </row.cell>
-          <row.cell>{{host.attributes.address}}</row.cell>
+          <row.cell>
+            {{#if host.isStatic}}
+              {{host.address}}
+            {{else}}
```

Comment:
@randallmorey 
My goal here was to leave an empty space `&nbsp`. So in case all the hosts in the list are plugin, the columns has content to not be shrinked.
But happy to eliminate the else clause and leave it empty when is plugin.
Let me know your thoughts.

Reply:
It's probably unnecessary.  I think the header cell would prevent collapse.  But not going to hold up merge.

Reply:
Indeed, Sorry I miss this comment, pushing that change.